b616.py

```diff
@@ -180,42 +180,42 @@ def draw_rt_sc_chart():
 
     lx_dt = min(x_detail)
     mx_dt = max(x_detail)
-    ptp_xdt = mx_dt - lx_dt
+    ptp_xdt = mx_dt - lx_dt  # ptp of numpy (不为这个特地import了)
     marksize = max(5, 10 - ptp_xdt - 0.03 * custom_num)  # 散点曲名标记的字体大小
 
     #  生成 rating/定数 图
     def rating2detail_chart():
-        lx_rt = min(y_rating)
+        ly_rt = min(y_rating)
         my_rt = max(y_rating)
-        ptp_yrt = my_rt - lx_rt
+        ptp_yrt = my_rt - ly_rt
         ################这一段是对图表进行初始化和自定义################
         fig, ax = plt.subplots()
         ax.scatter(x_detail, y_rating, s=(20 - pow(custom_num, 0.5)))
         ax.set_xlabel("谱面定数", fontsize=12)
         ax.set_ylabel("单曲Rating", fontsize=12)
         ax.axis(
-            [lx_dt - 0.01, mx_dt + 0.03, lx_rt - 0.01, my_rt + 0.02]
+            [lx_dt - 0.01, mx_dt + 0.05, ly_rt - 0.01, my_rt + 0.02]
         )  # 设置每个坐标轴的取值范围
         ax.tick_params(
             axis="both", which="major", labelright=True, labelsize=10, pad=2, color="r"
         )  # 设置刻度标记的样式
         ax.xaxis.set_major_locator(
             ticker.MultipleLocator(0.1)
-        )  # tick_spacing = 0.1: 横轴标注以0.1为单位步进
+        )  # tick_spacing = 0.1: 横轴标注以0.1为单位步进 (即arcaea官方定数的最小单位)
         ax.grid(
             axis="y", color="r", linestyle="--", linewidth=0.4
         )  # 设置图表的外观样式
         if custom_num >= 30:  # 生成理论ptt横线,图例自动放在最佳位置
             ax.axhline(
                 y=b30_withr10,
-                lw=1,
-                ls="-.",
+                lw=1, # linewidth
```

Comment:
just use `linewidth = 1` instead of `lw = 1`

Reply:
sure agreed